src/widgets/dragAndDrop.ts

```diff
@@ -8,34 +8,41 @@ module.exports = {
   makeDraggable: makeDraggable,
   uploadFiles: uploadFiles
 }
+
 // const UI = require('../index.js') // this package
 
-function makeDropTarget (ele, droppedURIHandler, droppedFileHandler) {
-  var dragoverListener = function (e) {
```

Comment:
should I add export on all the functions?

Reply:
Yes, see how @megoth did it in https://github.com/solid/solid-ui/pull/232